scripts/oneoff/generate_g8_master_csv.py

```diff
@@ -0,0 +1,72 @@
+"""
+Generate a CSV (to stdout) with per-lot draft statistics for each supplier
+
+Usage:
+    scripts/oneoff/generate_g8_master_csv.py <data_api_url> <data_api_token>
+
+Example:
+    scripts/oneoff/generate_g8_master_csv.py http://api myToken
+"""
+import csv
+from sys import stdout
+from itertools import product, chain
+from collections import Counter, OrderedDict
+
+from docopt import docopt
+from dmapiclient import DataAPIClient
+
+_lot_slug_status_label = lambda lot_slug, status: "{}_{}".format(lot_slug, status)
+
+
+def generate_g8_master_csv(
+        data_api_url,
+        data_api_token,
+        outfile=stdout,
+        target_framework_slug="g-cloud-8",
+        target_statuses_labels=OrderedDict((
+            ("submitted", "completed",),
+            ("not-submitted", "draft",),
+        )),
+        target_lot_slugs=("saas", "paas", "iaas", "scs",),
+        ):
+    client = DataAPIClient(data_api_url, data_api_token)
+
+    writer = csv.DictWriter(outfile, lineterminator="\n", fieldnames=(
+        "supplier_id",
```

Comment:
As the files are for CCS rather than developers, we could go with "nicer"* headings with capital letters and spaces.

* in the eye of the beholder, I know.